docassemble/EFSPIntegration/data/questions/any_filing_interview.yml

```diff
@@ -205,13 +204,66 @@ code: |
   if can_check_efile:
     if is_initial_filing:
       users[0].email
+    #TODO(qs): if not a waiver account...
```

Comment:
We might need to show the fees either way: there seem to be a handful of codes that let the court still charge for certain things with waiver payment accounts, like `allowwaieronmail` for the mail service type (not in IL though) and `allowwaiveronredaction` for redaction fees.